### PR TITLE
Fixing multiple fortify issues

### PR DESCRIPTION
**Pull request by duranmf**
https://tools.hmcts.net/jira/browse/CCD-3211
https://tools.hmcts.net/jira/browse/CCD-3212
https://tools.hmcts.net/jira/browse/CCD-3214

The Dockerfile sets a container to run with root user.
Retrieving build dependencies using a non-specific version can leave the build system vulnerable to malicious binaries or cause the system to experience unexpected behavior.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,10 @@
 # ---- Base Image ----
-ARG base=hmctspublic.azurecr.io/base/node:14-alpine
-
-FROM ${base} as base
+FROM hmctspublic.azurecr.io/base/node:14-alpine as base
 USER root
```
Comment (rajaram-nagarajan) [inline Dockerfile:3]: You can remove "USER root" from here as the default user is root

Comment (duranmf) [reply Dockerfile:3]: Removing the user root does not work, gets error after removing.

Comment (rajaram-nagarajan) [reply Dockerfile:3]: oh ok

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**